frontend/src/Admin/components/ShowSubmissionsTab.js

```diff
@@ -114,6 +150,35 @@ class ShowSubmissionsTab extends Component {
         >
           There was an error updating the invite status
         </Alert>
+        <Modal isOpen={this.state.modal}>
+          <ModalBody>
+            This is a permanent action. Clicking continue will make the invite
```

Comment:
"This is a permanent action and will make invitations for this show visible to all students."